src/AccountProfile.jsx

```diff
@@ -1,6 +1,7 @@
 const accountId = props.accountId || context.accountId;
 const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
 const profileUrl = `#/${REPL_ACCOUNT}/widget/ProfilePage?accountId=${accountId}`;
+const activityFeed = props.activityFeed;
```

Comment:
maybe this prop should be named something like `showActionButtons`?

Reply:
This prop defines the place where FlagButton is clicked. We need to be sure that flag account button is clicked on ActivityFeed

Reply:
Right. I was thinking we may show the AccountProfile lots of places. With some we'll want to show those buttons (like on the ActivityFeed), others we won't.

Reply:
Make sense

Reply:
ll